src/configureStore.ts

```diff
@@ -0,0 +1,9 @@
+import { createStore } from 'redux'
```

Comment:
Still a little empty this file but I guess there might come some more stuff. I know this goes in the direction of "premature optimization" but I like the structure.